File: python/tests/test_65.py
from python.src.code_65 import Solution
from python.tests.python_deps.boilerplate import BoilerTest


def input_fetcher(case):
    return case["input"]["s"]


def test():
    test = BoilerTest(65, input_fetcher)
    solution = Solution().isNumber
    test.run_tests(solution)

File: cpp/test/test_65.cpp
#include "../src/code_65.cpp"
#include "cpp_deps/boilerplate.hpp"

void test(Solution& sol, const json& input, const json& output) {
    string s = input["s"].get<string>();
    bool expected = output.get<bool>();
    bool result = sol.isNumber(s);
    CHECK_EQ(result, expected);
}

TEST_CASE("") {
    TEST(65);
}